services/core-api/app/api/securities/resources/bond.py

```diff
@@ -0,0 +1,88 @@
+from flask_restplus import Resource, reqparse
+from datetime import datetime
+from flask import current_app, request
+from werkzeug.exceptions import BadRequest, NotFound, InternalServerError
```

Comment:
InternalServerError is an unused import, can you remove it from this file and the others it's imported into but not being used?

Reply:
removed